pkg/crc/version/version.go

```diff
@@ -35,6 +35,9 @@ var (
 
 	// Bundle version which used for the release.
 	bundleVersion = "0.0.0-unset"
+
+	// Tray version to be embedded in binary
+	crcTrayVersion = "0.0.0-unset"
```

Comment:
I believe we could directly put 1.0.0-alpha here rather than passing it through linker flags, as we don't need CRC_TRAY_VERSION in the Makefile, only in the go code.

Reply:
Its kept in the makefile, for ease of updating the version of the tray, when a new version of CRC is released we bump the version in the Makefile and along with it we can bump the tray version too.

Reply:
For the machine drivers, the version is not in the Makefile, only in the go code, I'd be consistent here, this means less magic constants needed when building crc.

Reply:
OK, removed the version constant from Makefile.